hack/dockerfiles/fuzz.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
```

Comment:
Can we use `golang:1.15-alpine`

Reply:
Or maybe we can add `fuzz` stage o the top-level Dockerfile

Reply:
Gofuzz has some issues with Go 1.15 which is also the reason go 1.14.9 is installed.

Reply:
go-fuzz breaks with Go 1.15: https://github.com/dvyukov/go-fuzz/issues/294
This is also the reason Go 1.14.9 is installed.

Reply:
dvyukov/go-fuzz#294 seems closed